docs/running-workflows/launching-workflows/index.md

```diff
@@ -135,22 +135,13 @@ The standard set of badges look as follows:
 
 ![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
 
-If you would like to generate your own custom badge, you can use external services such as [shields.io](https://shields.io) or [badgen.net](https://badgen.net).
+To streamline the badge creation process, you can use the tool accessible on REANA at the
```

Comment:
What if we keep the sections about Markdown and HTML, and add a new one before those for the badge creator? This way we can show both the "automatic" way to generate badges, and the "manual" way to do it with Markdown and HTML.

What do you think?